--- src/include.orig/epsilon_private.h	2006-03-28 03:10:14.000000000 -0600
+++ src/include/epsilon_private.h	2007-01-20 22:35:54.254465050 -0600
@@ -41,7 +41,7 @@
 Epsilon_Message *epsilon_message_new(int clientid, char *path, char *dst, int status);
 
 int epsilon_ipc_client_send(Ecore_Con_Client *cl, Epsilon_Message *msg);
-int epsilon_ipc_server_send(Ecore_Con_Client *cl, Epsilon_Message *msg);
+int epsilon_ipc_server_send(Ecore_Con_Server *srv, Epsilon_Message *msg);
 
 Epsilon_Message *epsilon_ipc_consume(Epsilon_Ipc_End *end);
 
